pkg/importer/spanner/import.arrai

```diff
@@ -0,0 +1,14 @@
+let [_, importSql, sqlSchema, syslPackage] = //os.args;
```

Comment:
Would `appName` be more appropriate than `sqlSchema`?

Taking the following description of parameters from the ticket:
```
-a, --app-name=<APP-NAME> The name of the Sysl app.
Required if the schema doesn't include a CREATE DATABASE statement.
-p, --package=<PACKAGE> The name of the Sysl package.
```

* `appName` is optional. This should overwrite the existing value if provided or return an error if not provided and required.
* `syslPackage` is optional. This shouldn't be added to the output if not provided.

Reply:
yes ok